docs/contrib/intellij.rst

```diff
@@ -0,0 +1,17 @@
+.. _intellij:
+
```

Comment:
After renaming the file, please add a section about Metals. A simple "Metals import should work out of the box for most of the modules." would be enough.